packages/gestalt-datepicker/src/DatePicker.css

```diff
@@ -0,0 +1,310 @@
+  position: absolute;
+  top: 0;
+  width: 100%;
+}
+
+:global ._gestalt .react-datepicker__day--range-end:hover {
+  background-color: inherit;
+  border-radius: var(--borderRadius-halfCircle-Start);
+  color: var(--color-darkGray);
+  position: relative;
+}
+
+:global ._gestalt .react-datepicker__day--range-end:hover::after {
+  background: var(--color-hover-lightGray);
+  border-radius: var(--borderRadius-circle);
+  content: var(--empty-content);
+  display: block;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  width: 100%;
+}
+
+:global ._gestalt .react-datepicker__day--selecting-range-start:not(.react-datepicker__day--in-range),
+:global ._gestalt .react-datepicker__day--selecting-range-end:not(.react-datepicker__day--in-range) {
+  background-color: var(--color-hover-lightGray);
+}
+
+:global ._gestalt .react-datepicker__day--outside-month {
+  visibility: hidden;
+}
+
+:global ._gestalt .react-datepicker__day-name {
+  color: var(--color-gray);
+  display: inline-block;
+  font-size: 16px;
+  line-height: var(--day-dimensions);
+  margin: 0;
+  text-align: center;
+  width: var(--day-dimensions);
+}
+
+:global ._gestalt .react-datepicker__day-names {
+  display: flex;
+  white-space: nowrap;
+}
+
+:global ._gestalt .react-datepicker__header {
+  position: relative;
+  text-align: center;
+}
+
+:global ._gestalt .react-datepicker__input-container {
+  display: block;
+  position: relative;
+  width: 100%;
+}
+
+:global ._gestalt .react-datepicker__input-container input {
+  padding-right: 38px;
+}
+
+:global ._gestalt .react-datepicker__month {
+  color: var(--color-white);
+  text-align: center;
+}
+
+:global ._gestalt .react-datepicker__week {
+  display: flex;
+  white-space: nowrap;
+}
+
+:global ._gestalt .react-datepicker__month-container {
+  float: left;
+  font-size: 16px;
+  font-weight: normal;
+}
+
+:global ._gestalt .react-datepicker__navigation {
+  background: none;
+  border: none;
+  border-radius: var(--borderRadius-circle);
+  cursor: pointer;
+  height: auto;
+  outline: none;
+  overflow: visible;
+  padding: 6px;
+  position: absolute;
+  top: 14px;
+  width: auto;
+  z-index: 1;
+}
+
+:global ._gestalt .react-datepicker__navigation--previous {
+  left: 8px;
+}
+
+:global ._gestalt .react-datepicker__navigation--previous--disabled,
+:global ._gestalt .react-datepicker__navigation--previous--disabled:hover {
+  cursor: default;
+}
+
+:global ._gestalt .react-datepicker__navigation--next {
+  right: 8px;
+}
+
+:global ._gestalt .react-datepicker__navigation--next--disabled,
+:global ._gestalt .react-datepicker__navigation--next--disabled:hover {
+  cursor: default;
+}
+
+:global ._gestalt .react-datepicker__navigation:focus {
+  box-shadow: var(--focus);
+}
+
+.react-datepicker-popper {
+  z-index: 1;
+}
+
+.react-datepicker-popper[data-placement^="bottom"] {
+  margin-top: 6px;
+}
+
+.react-datepicker-popper[data-placement^="top"] {
+  /* !important is required to keep same distance from Textfield
```

Comment:
@AlbertCarreras let's avoid this, could you use a different hack instead?

`.react-datepicker-popper.react-datepicker-popper[data-placement^="top"]` could also be a potential fix (without using `!important`

Reply:
It does work
![image](https://user-images.githubusercontent.com/10593890/84451602-702dff80-ac08-11ea-803d-31715a38ce15.png)
with !important
![image](https://user-images.githubusercontent.com/10593890/84451655-8b990a80-ac08-11ea-8a22-dc94272002ce.png)
with .react-datepicker-popper.react-datepicker-popper[data-placement^="top"]
![image](https://user-images.githubusercontent.com/10593890/84451792-e894c080-ac08-11ea-89bc-302873e352db.png)